README.md

```diff
@@ -36,8 +36,9 @@ Clone repository
 
     git clone https://github.com/oat-sa/package-tao.git
     
-Install via composer missing library and extensions
+Install via composer missing library and extensions. Composer version 2 doesn't work so you've to use version 1.
 
+    composer self-update --1
```

Comment:
maybe we don't this here as we have have it in the (https://www.taotesting.com/user-guide/installation-and-upgrade/prerequisites/)

Reply:
fixed